PowerEditor/installer/nativeLang/korean.xml

```diff
@@ -1,1418 +1,1418 @@
+                    <Item subMenuId="edit-blankOperations" name="공백 기능(&amp;B)"/>
+                    <Item subMenuId="edit-pasteSpecial" name="붙여넣기 특수기능(&amp;P)"/>
+                    <Item subMenuId="edit-onSelection" name="선택시(&amp;O)"/>
+                    <Item subMenuId="search-markAll" name="모든 토큰에 형식 적용(&amp;A)"/>
+                    <Item subMenuId="search-markOne" name="한 개 토큰에 형식 적용(&amp;O)"/>
+                    <Item subMenuId="search-unmarkAll" name="형식 해제"/>
+                    <Item subMenuId="search-jumpUp" name="위로 이동(&amp;J)"/>
+                    <Item subMenuId="search-jumpDown" name="아래로 이동(&amp;D)"/>
+                    <Item subMenuId="search-copyStyledText" name="형식을 가진 텍스트 복사(&amp;C)"/>
+                    <Item subMenuId="search-bookmark" name="책갈피(&amp;B)"/>
+                    <Item subMenuId="view-currentFileIn" name="현재 파일 보기 위치"/>
+                    <Item subMenuId="view-showSymbol"  name="기호 표시"/>
+                    <Item subMenuId="view-zoom"  name="확대/축소"/>
+                    <Item subMenuId="view-moveCloneDocument"  name="현재 문서 이동/복제"/>
+                    <Item subMenuId="view-tab"  name="탭"/>
+                    <Item subMenuId="view-collapseLevel" name="단계 접기"/>
+                    <Item subMenuId="view-uncollapseLevel" name="단계 펼치기"/>
+                    <Item subMenuId="view-project" name="프로젝트"/>
+                    <Item subMenuId="encoding-characterSets"  name="문자 집합"/>
+                    <Item subMenuId="encoding-arabic"  name="아랍어"/>
+                    <Item subMenuId="encoding-baltic"  name="발트어"/>
+                    <Item subMenuId="encoding-celtic"  name="켈트어"/>
+                    <Item subMenuId="encoding-cyrillic"  name="키릴어"/>
+                    <Item subMenuId="encoding-centralEuropean"  name="중유럽어"/>
+                    <Item subMenuId="encoding-chinese"  name="중국어"/>
+                    <Item subMenuId="encoding-easternEuropean"  name="동유럽어"/>
+                    <Item subMenuId="encoding-greek"  name="그리스어"/>
+                    <Item subMenuId="encoding-hebrew"  name="히브리어"/>
+                    <Item subMenuId="encoding-japanese"  name="일본어"/>
+                    <Item subMenuId="encoding-korean" name="한국어"/>
+                    <Item subMenuId="encoding-northEuropean" name="북유럽어"/>
+                    <Item subMenuId="encoding-thai" name="태국어"/>
+                    <Item subMenuId="encoding-turkish" name="터키어"/>
+                    <Item subMenuId="encoding-westernEuropean" name="서유럽어"/>
+                    <Item subMenuId="encoding-vietnamese" name="베트남어"/>
+                    <Item subMenuId="language-userDefinedLanguage" name="사용자 정의 언어"/>
+                    <Item subMenuId="settings-import" name="가져오기"/>
+                    <Item subMenuId="tools-md5" name="MD5"/>
+                    <Item subMenuId="tools-sha256" name="SHA-256"/>
+                    <Item subMenuId="window-sortby" name="정렬 기준"/>
+                </SubEntries>
```

Comment:
There should not be the diff between these 2 blocks. 
The change of formatting of file is not accepted, since it makes review difficult.